test/dbus.c

```diff
@@ -740,6 +815,45 @@ TEST test_signal_actioninvoked(void)
         PASS();
 }
 
+TEST test_signal_length_propertieschanged(void)
+{
+        struct dbus_notification *n_dbus;
+        struct signal_propertieschanged sig = {NULL, NULL, NULL, -1};
+
+        dbus_signal_subscribe_propertieschanged(&sig);
+
+        n_dbus = dbus_notification_new();
+        n_dbus->app_name = "dunstteststack";
+        n_dbus->app_icon = "NONE2";
+        n_dbus->summary = "Headline for New";
+        n_dbus->body = "Text";
+        g_hash_table_insert(n_dbus->actions, g_strdup("actionkey"), g_strdup("Print this text"));
+
+        guint id;
+        ASSERT(dbus_notification_fire(n_dbus, &id));
+        ASSERT(id != 0);
+
+        signal_length_propertieschanged();
```

Comment:
The point of this test is to check if the signal gets sent automatically, right? So you should use something like `queues_update` and check if it sends the signal.

Reply:
Yes, I think `queues_update` is the best place to put it.